.coderabbit.yaml

```
@@ -0,0 +1,92 @@
+language: ko-KR
+
+tone_instructions: |
+  - 팀의 Frontend Design Guideline을 우선 참고해요.
+  - 런타임에 에러가 발생할 수 있는 부분 또는 의도치 않은 동작이 있는지 검사해요.
+  - 패턴 제시보다는 "왜" 개선이 필요한지 먼저 설명해요.
+
+reviews:
+  profile: assertive # 리뷰 스타일 설정 - chill(일반), assertive(꼼꼼)
```

YAMLlint (1.37.1)

[warning] too few spaces before comment: expected 2

(comments)